example/spec/parallel/testAPI/oneTimeRule.js

```diff
@@ -53,36 +48,8 @@ function removeRuleAddedParams(rule) {
   return ruleCopy;
 }
 
-/**
- * Wait for the execution kicked off by the rule to begin.
- * We check that the execution input has the rule name in meta.triggerRule
- * so that we know we are looking at the right execution of the workflow
- * and not one that could have been triggered by something else
- *
- * @param {string} - string to match triggerRule
- * @returns {undefined} - none
- */
-async function waitForTestExecution(executionMatch) {
-  let timeWaitedSecs = 0;
-  let workflowExecution;
-
-  /* eslint-disable no-await-in-loop */
-  while (timeWaitedSecs < maxWaitForStartedExecutionSecs && workflowExecution === undefined) {
-    await sleep(waitPeriodMs);
-    timeWaitedSecs += (waitPeriodMs / 1000);
-    const executions = await getExecutions(helloWorldRule.workflow, config.stackName, config.bucket);
-
-    for (const execution of executions) {
-      const taskInput = await lambdaStep.getStepInput(execution.executionArn, 'SfSnsReport');
-      if (taskInput && taskInput.meta.triggerRule && taskInput.meta.triggerRule === executionMatch) {
-        workflowExecution = execution;
-        break;
-      }
-    }
-  }
-  /* eslint-disable no-await-in-loop */
-  if (timeWaitedSecs < maxWaitForStartedExecutionSecs) return workflowExecution;
-  throw new Error('Never found started workflow.');
+function compareTriggerRule(taskInput, params) {
```

Comment:
while I'm going on about names,  something like `compareTriggerRules` is more intuitive to me (because it looks like triggerRules from two different sources are being compared).

Reply:
Changed this to `isWorkflowTriggeredByRule`